tests/test_parameter_generation.py

```diff
@@ -439,10 +439,12 @@ def test_cpm_sm_data_can_be_fit_successively(datasets_db):
     datasets_db.insert(CU_ZN_SM_MIX_EXPR_TO_FLOAT)
     dbf = generate_parameters(CU_ZN_LIQUID_PHASE_MODEL, datasets_db, 'SGTE91', 'linear')
     # beware that the calculate() results will not match up exactly with the original data due to rounding of parameters
-    assert dbf.symbols['VV0000'] == 105.255  # T*ln(T) L2 term
-    assert dbf.symbols['VV0001'] == -40.953  # T*ln(T) L1 term
-    assert dbf.symbols['VV0002'] == -44.57 # T*ln(T) L0 term
-    assert dbf.symbols['VV0003'] == 36.6556 # L0 T term, found after CPM_MIX addition
+
+    assert dbf.symbols['VV0000'] == 36.6556  # L0 T term, found after CPM_MIX addition
+    assert dbf.symbols['VV0001'] == 105.255  # T*ln(T) L2 term
+    assert dbf.symbols['VV0002'] == -40.953  # T*ln(T) L1 term
+    assert dbf.symbols['VV0003'] == -44.57 # T*ln(T) L0 term
```

Comment:
@richardotis are the ordering changes here due to a change in the sorting of T-dependent terms from the pycalphad side?

Reply:
Yes. The terms were ordered by their string representation, and it appears that SymEngine orders the terms slightly differently from SymPy.

Reply:
My preference is for this ordering (it was always a nit of mine in the way they were sorted). This may be a breaking corner case in some existing workflows that depend on a particular ordering.

I don't think it blocks this issue. I might actually use the breaking change as an opportunity to reorder the numbering to go from L0...L4 rather than L4...L0.

Reply:
This order could be controlled directly by writing a sorting function that explicitly sets the exponent of each variable. In principle this change breaks the determinism of any existing end-to-end workflow, though previously generated TDBs should give the same result in the MCMC step. It depends on what sort of guarantees you're looking to provide here.

Reply:
I wrote a sorting function (`stable_sort_key`) that seems to reproduce the old ordering, so I reverted the changes to the test.